keyboards/rico/phoenix_project_no1/info.json

```diff
@@ -0,0 +1,123 @@
+{
+    "usb": {
+        "vid": "0x21C0",
+        "pid": "0x9901",
+        "device_version": "1.0.0"
+    },
+    "keyboard_name": "Phoenix Project No 1",
+    "manufacturer": "Rico",
+    "url": "https://github.com/mymakercorner/Phoenix_Project_No1",
+    "maintainer": "Rico",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "board": "GENERIC_RP_RP2040",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false,
+        "rgblight": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["GP24", "GP0", "GP14", "GP13", "GP12", "GP10", "GP9", "GP8", "GP7", "GP6", "GP5", "GP4", "GP3", "GP25", "GP2", "GP1"],
+        "rows": ["GP27", "GP29", "GP19", "GP20", "GP26"]
+    },
+    "ws2812": {
+        "pin": "GP28",
+        "driver": "vendor"
+    },
+    "rgblight": {
+        "led_count": 16,
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        }
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "matrix": [0, 0], "x": 0, "y": 0 },
```

Comment:
Hello fauxpark,

It has been a long time since I resolved this issue.
Could you please validate(or not) that this issue has been fixed ?

Thanks in advance ! 

Best Regards,

Rico